### DISCUSSION TITLE
Gullfoss VST doesnt work

### DISCUSSION THREAD

**Discussion by sammbusy** [apohl79/audiogridder#304] [Help]
Hello guys, I tried the Gullfoss plugin with my Parallels virtual machine running windows, through my AudioGridder on my MAC (Reaper).

Plugins work fine, except one, the Gullfoss... it seems it doesnt affect the audio.

Anyone has this problem?

**Comment by CaBuffalo** [comment]
Let me test it tonight.

**Comment by sammbusy** [comment]
tested on 1.4.1 but i think its just the plugin that doesnt work, tried with 1.4.0 and works normal i guess. thanks anyway

**Reply by CaBuffalo** [reply]
OK, sounds good. Thanks.
